app/cypress/fixtures/documentTests.js

```diff
@@ -1,8 +1,8 @@
-import minimalDoc from '../shared/minimalCSAFBaseDoc'
-import minimalInformationalAdvisoryDoc from '../shared/minimalInformationalAdvisoryDoc'
-import minimalSecurityAdvisoryDoc from '../shared/minimalSecurityAdvisoryDoc'
-import minimalSecurityIncidentResponseDoc from '../shared/minimalSecurityIncidentResponseDoc'
-import minimalVexDoc from '../shared/minimalVexDoc'
+import minimalDoc from './documentTests/minimalGenericCSAFDoc'
+import minimalInformationalAdvisoryDoc from './documentTests/minimalInformationalAdvisoryDoc'
+import minimalSecurityAdvisoryDoc from './documentTests/minimalSecurityAdvisoryDoc'
+import minimalSecurityIncidentResponseDoc from './documentTests/minimalSecurityIncidentResponseDoc'
+import minimalVexDoc from './documentTests/minimalVexDoc'
```

Comment:
Why don't we need the `.js` here? We needed it in all other files...

Reply:
Good catch! It seems that the webpack bundled with cypress doesn't care about the `"type": "module"` in the package.json. But I'll keep it consistent.

Reply:
Done in [3e5687f](https://github.com/secvisogram/secvisogram/pull/241/commits/3e5687f006764a7938f8d747c8d860d47e732920).